### ISSUE TITLE
Prometheus: Cannot draw polygon fuel patch

### ISSUE THREAD

**Issue by nealmcloughlin** [PSaaS-Developers/Project_issues#119]
Polygon fuel patches can no longer be drawn on the map view of Prometheus version 2021.12.03. File imports work, but I can't manually draw a polygon.

**Comment by RobBryce** [comment]
I just tried to recreate this so I could begin to draft a plan here.  I can create fuel patches without issue, and they modify the fire growth accordingly.

**Comment by spydmobile** [comment]
@RobBryce Can you confirm if this problem exists/does not in the reported version? The bug, you are chasing was reported in 12.03 Does it exist in that version? @nealmcloughlin  If not I am not sure what we do with this.

**Comment by spydmobile** [comment]
Ok, @RobBryce  lets confirm the bug PRESENT in the 12.03 version and that it is NOT PRESENT in the current version. I am not a fan of mystery bugs appearing and disappearing without explanation, but if this is the case, I think we will close this as fixed in current version. Just let me know.

**Comment by nealmcloughlin** [comment]
This is really weird. I experienced this bug back in July will using version 2021.12.03. I just tried it again with the same version and the bug has mysteriously fixed itself. I know I'm not crazy because I needed to replace non-fuels where a fire I was modeling had started. The only way I could do so was by drawing an ignition polygon, exporting it, and then importing the polygon back in as a fuel patch. Here's what did change between then and now. At the time my REDapp install was working. Now my REDapp install will not open. Anyways, I'll close this as I can no longer replicate it either.
